frontend/packages/dev-console/src/components/topology/TopologyEdgePanel.tsx

```diff
@@ -32,17 +41,25 @@ const connectorTypeToTitle = (type: string): string => {
       return 'Traffic distribution connector';
     case TYPE_EVENT_SOURCE_LINK:
       return 'Event source connector';
+    case TYPE_TRAFFIC_CONNECTOR:
+      return 'Traffic connector';
     default:
       return '';
   }
 };
 
-const TopologyEdgePanel: React.FC<TopologyEdgePanelProps> = ({ edge, data }) => {
+export const TopologyEdgePanel: React.FC<TopologyEdgePanelProps> = ({
```

Comment:
Is there any reason we want to export this on its own? I believe no one uses it but the one spot you are using the export default.

Reply:
Yeah will remove it but later for testing it might be used elsewhere